ci/prow/config_knative.yaml

```diff
@@ -100,6 +100,11 @@ presubmits:
 periodics:
   knative/serving:
     - continuous: true
+      istio: "1.0.7"
+      cron: "1 */2 * * *" # Run every other hour and one minute
```

Comment:
Better distribute the jobs for better resource usage. Once day this should be automatic, but right now just use a different minute.